relational_operators/CreateIndexOperator.hpp

```diff
@@ -75,7 +83,8 @@ class CreateIndexOperator : public RelationalOperator {
 
  private:
   CatalogRelation *relation_;
-  const std::string &index_name_;
+  std::string index_name_;
```

Comment:
@zuyu I was wondering that whether initializing a private member string with a reference should be recommended or not. Because lets say in the distributed version, what if the referenced string changes between the time the CreateIndexOperator is marked for execution by Foreman and it is actually executed? Is it okay to use a reference in such cases?